src/api/layers.api.js

```diff
@@ -190,24 +190,29 @@ const generateClassForLayerConfig = (layerConfig) => {
  */
 const loadLayersConfigFromBackend = (lang) => {
     return new Promise((resolve, reject) => {
-        const layersConfig = [];
-        axios.get(`${API_BASE_URL}rest/services/all/MapServer/layersConfig?lang=${lang}`)
-            .then(({data: rawLayersConfig}) => {
-                if (Object.keys(rawLayersConfig).length > 0) {
-                    Object.keys(rawLayersConfig).forEach(rawLayerId => {
-                        const rawLayer = rawLayersConfig[rawLayerId];
-                        const layer = generateClassForLayerConfig(rawLayer);
-                        if (layer) layersConfig.push(layer)
-                    })
-                    resolve(layersConfig);
-                } else {
-                    reject('LayersConfig loaded from backend is not an defined or is empty');
-                }
-            }).catch((error) => {
+        if (!API_BASE_URL) {
+            // this could happen if we are testing the app in unit tests, we simply reject and do nothing
+            reject('API base URL is undefined');
+        } else {
+            const layersConfig = [];
+            axios.get(`${API_BASE_URL}rest/services/all/MapServer/layersConfig?lang=${lang}`)
+                .then(({data: rawLayersConfig}) => {
+                    if (Object.keys(rawLayersConfig).length > 0) {
+                        Object.keys(rawLayersConfig).forEach(rawLayerId => {
+                            const rawLayer = rawLayersConfig[rawLayerId];
+                            const layer = generateClassForLayerConfig(rawLayer);
+                            if (layer) layersConfig.push(layer)
```

Comment:
Code style: I'm not a big fan of inline if ;-)

Reply:
yep, this one slip through, thanks for the spotting